defaults/main.yml

```diff
@@ -10,3 +10,5 @@ varnish_config_path: /etc/varnish
 varnish_admin_listen_host: "127.0.0.1"
 varnish_admin_listen_port: "6082"
 varnish_storage: "file,/var/lib/varnish/varnish_storage.bin,256M"
+varnish_root_group: root
+varnish_service_name: varnish
```

Comment:
Having this defined in defaults might mean the service name `varnish` would still be used unless someone on a FreeBSD server overrode the variable in the playbook using this role.

This variable (and the root group) should probably follow the pattern here, with the double-underscore included variable: https://github.com/geerlingguy/ansible-role-apache/blob/master/vars/RedHat.yml#L10